packages/compass-components/src/components/modals/modal-header.tsx

```diff
@@ -57,32 +46,62 @@ const warningIconStyles = css({
   },
 });
 
-const warningIconStylesDark = css({
-  background: `${palette.red.dark2}`,
-});
+const variantLightStyles = {
```

Comment:
Some thoughts: what are we trying to do here and for which purpose? Are the current components and styles not enough for this niche use-case? Are we sure we shouldn't negotiate a compromise with design?

Reply:
currently in modal header we only support two variants (default - with no icon and danger - with red filled warning icon) and the design came up with a warn variant (which uses yellow) as fill color. 

I did try to use danger variant to avoid these changes, but Ben suggested to use warn variant.

Reply:
I did have another chat with Ben and he agreed to drop this. I'll make the changes.

Reply:
removed!